.rubocop.yml

```diff
@@ -26,7 +26,7 @@ Metrics/ClassLength:
   Max: 150
 Metrics/BlockLength:
   IgnoredMethods: ['describe']
-  Max: 30
+  Max: 31
```

Comment:
[Optional]
- [ ] Please do not change the rubocop configuration. You should try to solve the problems. If the solution is too difficult, you can suppress checks for the problem for the file or code block only.